File: api/golang/engine/lib/kurtosis_context/service_log.go
package kurtosis_context

//This is an object to represent a simple log line information
type ServiceLog struct {
	//lineTime time.Time //TODO add the time from loki logs result
	content string
}

func newServiceLog(content string) *ServiceLog {
	return &ServiceLog{content: content}
}

func (serviceLog ServiceLog) GetContent() string {
	return serviceLog.content
}

File: api/typescript/src/engine/lib/kurtosis_context/service_log.ts
//This is an object to represent a simple log line information
export class ServiceLog {
    //private readonly lineTime: Date; //TODO add the time from loki logs result
    private readonly content: string;

    constructor(content: string) {
        this.content = content;
    }

    public getContent():string {
        return this.content;
    }
}
